cmd/ef_tests/state/Makefile

```
@@ -159,3 +159,8 @@ samply-benchmarks: ## ⚡️ Run benchmarks and create samply profiling file
 	$(call run_samply,ERC20Approval,REPETITIONS_SLOW,BENCH_APPROVAL_ITERATIONS)
 	$(call run_samply,ERC20Transfer,REPETITIONS_SLOW,BENCH_TRANSFER_ITERATIONS)
 	$(call run_samply,ERC20Mint,REPETITIONS_SLOW,BENCH_MINT_ITERATIONS)
+
+## New EF state tests runner
+TESTS_PATH := ./vectors/GeneralStateTests
```

```suggestion
TESTS_PATH := ./vectors
```
Doesn't it work with all of them?

---

Yes it does! [Fixed here](https://github.com/lambdaclass/ethrex/pull/3642/commits/ab66cbce6cdeeadff888053d2a18a86ea66e6068)